model/QueueBroker/SqsQueueBroker.php

```diff
@@ -190,10 +190,34 @@ protected function doPop()
             $this->createQueue();
         }
 
-        $logContext = [
-            'QueueUrl' => $this->queueUrl
-        ];
+        $logContext = ['QueueUrl' => $this->queueUrl];
```

Comment:
why are we touching this ? 
`SqsQueueBroker which is for using AWS SQS.` -- and this will be a google cloud platform solution

Reply:
As stated in the commit, this was just to have an homogenous behavior between the SQS and RDS code. But I can leave it out if you like.